src/js/components/search/filters/recipient/RecipientResults.jsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * RecipientResults.jsx
+ * Created by Brian Petway 10/01/2024
+ */
+
+import React, { useEffect, useState } from 'react';
+import PropTypes from "prop-types";
+import * as SearchHelper from 'helpers/searchHelper';
+import PrimaryCheckboxType from 'components/sharedComponents/checkbox/PrimaryCheckboxType';
+
+const propTypes = {
+    selectedRecipients: PropTypes.object,
+    toggleRecipient: PropTypes.func
+};
+
+const RecipientResults = ({ toggleRecipient }) => {
+    const [recipients, setRecipients] = useState([]);
+
+    const levelMapping = {
+        P: 'Parent',
+        R: 'Recipient',
+        C: 'Child'
+    };
+
+    useEffect(() => {
+        if (recipients.length === 0) {
```

Comment:
We should check for inflight calls and cancel them to prevent an excess api calls.

Reply:
Also once this is working with the input field, this code here could create an infinite loop.  I suggest moving the fetch out of the use effect.  right now the fetch only needs to happen when the component is first loaded.

Reply:
good catch. is this what you had in mind?

Reply:
yes looks good!